third_party/intellij-plugins-dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcessZ.java

```diff
@@ -621,4 +607,61 @@ private static String threeSlashize(@NotNull final String uri) {
     if (uri.startsWith("file:")) return "file:///" + uri.substring("file:".length());
     return uri;
   }
+
+  private boolean baseUriInited = false;
```

Comment:
Makes sense! I moved the field definition, renamed it a bit, and will look at opening the access level in the parent class.